src/Components/common/Modal/ModalFolderItem.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  width: 264px;
+  border-radius: 8px;
+  padding: 8px;
+  gap: 8px;
+  font-family: Pretendard;
+  align-items: center;
+  position: relative;
+`;
+
+const FocusContainer = styled(Container)`
+  background: var(--Linkbrary-bg, #f0f6ff);
+`;
+
+const FolderName = styled.h2`
+  color: var(--Linkbrary-gray100, #373740);
+  font-size: 16px;
+  font-weight: 400;
+`;
+
+const FolderLink = styled.p`
+  color: var(--Linkbrary-gray60, #9fa6b2);
+  font-size: 14px;
+  font-weight: 400;
+`;
+
+const FocusFolderName = styled(FolderName)`
+  color: var(--primary-color, #6d6afe);
+`;
+
+const CheckIcon = styled.img`
+  position: absolute;
+  right: 8px;
+`;
+
+function ModalFolderItem({ folder, selected, onClick }) {
+  const { name, link } = folder;
+
+  return selected ? (
+    <FocusContainer onClick={onClick}>
+      <FocusFolderName>{name}</FocusFolderName>
+      <FolderLink>{link.count}개 링크</FolderLink>
+      <CheckIcon src="/Icons/check.png" alt="check" />
+    </FocusContainer>
+  ) : (
+    <Container onClick={onClick}>
+      <FolderName>{name}</FolderName>
+      <FolderLink>{link.count}개 링크</FolderLink>
+    </Container>
+  );
```

Comment:
@78-artilleryman 

이부분도 prop을 배리언스로 전달받와 활용해보면 어떨까요?

```suggestion
    <Container focused={selected} onClick={onClick}>
      <FolderName>{name}</FolderName>
      <FolderLink>{link.count}개 링크</FolderLink>
      { selected &&   <CheckIcon src="/Icons/check.png" alt="check" />}
    </Container>
```